### DISCUSSION TITLE
Require posts to be published in the future

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-checklists#437] [Feature Requests]
Label: FEATURE REQUEST

User story: 
As a user, I want to have the ability to require my writer to be published the post ALWAYS in the future. I always scheduled my post instead of publishing it directly. Sometimes I forgot to do so and inadvertently directly publish it.

Solution:
Introduce checklist to always publish a post in the future

https://wordpress.org/support/topic/require-posts-to-be-published-in-the-future-never-the-past/#post-15911548

**Comment by stevejburge** [comment]
Solved in 2.19
